client/app/components/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,110 @@
+class NavbarNotificationService {
+    constructor ($interval, $q, $translate, Alerter, constants, OvhApiNotificationAapi, UNIVERSE) {
+        this.$interval = $interval;
+        this.$q = $q;
+        this.$translate = $translate;
+        this.alerter = Alerter;
+        this.constants = constants;
+        this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+        this.UNIVERSE = UNIVERSE;
+
+        this.NOTIFICATION_REFRESH_TIME = 60000;
+    }
+
+    getMessages () {
+        return this.$translate.refresh().then(() => this.OvhApiNotificationAapi.query({
+            lang: this.$translate.preferredLanguage(),
+            target: this.constants.target,
+            universe: this.UNIVERSE
+        }).$promise.catch((error) => {
```

Comment:
No need for this, `throw` in `catch` will go through next `catch`

Reply:
I think `return ...$promise` is not very clear.
What's missing in this block is a way to log the error with sentry, do you know the good way of doing that?

Reply:
Then you should change the `catch` in `ovh-api-services` so that it logs to Sentry every time 😃